test/e2e/set/scyllacluster/scyllacluster_scaling.go

```diff
@@ -57,6 +69,10 @@ var _ = g.Describe("ScyllaCluster", func() {
 
 		verifyScyllaCluster(ctx, f.KubeClient(), sc)
 
+		framework.By("Updating data")
+		err = l.Update()
```

Comment:
if we scale, do we need to update the replication factor? could we just use a new instance of DataInseter which would always set it correctly based on the updated ScyllaCluster?

Reply:
removed Update method. Creating new object and calling Write() again as an alternative